File: app/src/main/java/com/code/runner/MainActivity.java
package com.code.runner;

import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;
import android.widget.EditText;
import android.widget.LinearLayout;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        LinearLayout layout = new LinearLayout(this);
        layout.setOrientation(LinearLayout.VERTICAL);

        EditText editText = new EditText(this);
        editText.setHint("Enter text here");

        layout.addView(editText);

        setContentView(layout);
    }
}

File: app/src/main/java/com/code/runner/MainActivity.kt
package com.code.runner

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.code.runner.Editor
class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        if(savedInstanceState == null) {
            supportFragmentManager.beginTransaction()
                .replace(R.id.fragment_container, Editor.newInstance())
                .commit()
        }
    }
}